### ISSUE TITLE
Pulumi console does not show complete output for errors in Powershell console

### ISSUE THREAD

**Issue by Rookian** [pulumi/pulumi#4188]
When I run pulumi up and I retrieve errors, they are not displayed completely. 

![image](https://user-images.githubusercontent.com/422617/77669129-a9998c00-6f84-11ea-9c5a-071e9db43fcb.png)

Pulumi.Azure 2.2.0-preview 
Pulumi 1.12.1-preview

**Comment by leezen** [comment]
The info column will truncate, but you should see any error/warning messages at the end of the run. Do those not appear in the `Diagnostics` section at the end of the run?

**Comment by Rookian** [comment]
> The info column will truncate, but you should see any error/warning messages at the end of the run. Do those not appear in the `Diagnostics` section at the end of the run?

I am not sure if pulumi is still running or if it crashed in this case. 

But I have a new case where I first had Azure Components with only logical names and then I'd like to migrate them to have physical names.

![image](https://user-images.githubusercontent.com/422617/77738790-b8c61b80-7010-11ea-84c9-1f6c61d4a8b6.png)

Should there be a summary displayed or did pulumi crashed? I am right now in an incosistent state.

**Comment by leezen** [comment]
@Rookian thanks for the additional details. Would you mind sharing your program or some de-sensitized version of it?

**Comment by Rookian** [comment]
```csharp
using Pulumi;
using Pulumi.Azure.AppInsights;
using Pulumi.Azure.AppService;
using Pulumi.Azure.AppService.Inputs;
using Pulumi.Azure.Search;
using Pulumi.Azure.Sql;

namespace Repro
{
    public class MyStack : Stack
    {
        public MyStack()
        {
            AzureSearch();
            AddSqlServer();
            RawPriceImportStorage();
            AzureSearchApps();
        }

        private void AzureSearchApps()
        {
            var appServicePlan = new Plan("proxxx-pocwebs-win", new PlanArgs
            {
                // Name = "proxxx-pocwebs-win",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                Kind = "App",
                Sku = new PlanSkuArgs
                {
                    Tier = "Basic",
                    Size = "B1",
                }
            });

            
            var insightsForSearchApi = new Insights("proxxxsearchapi", new InsightsArgs
            {
                // Name = "proxxxsearchapi",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                ApplicationType = "web"
            });

            var insightsForSearchUi = new Insights("proxxxprodsearch", new InsightsArgs
            {
                // Name = "proxxxprodsearch",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                ApplicationType = "web"
            });

            new AppService("proxxxsearchapi-poc", new AppServiceArgs
            {
                // Name = "proxxxsearchapi-poc",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                AppServicePlanId = appServicePlan.Id,
                AppSettings =
                {
                    { "ApplicationInsights:InstrumentationKey", insightsForSearchApi.InstrumentationKey }
                },
                HttpsOnly = true
            });

            new AppService("proxxxproductsearch", new AppServiceArgs
            {
                // Name = "proxxxproductsearch",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                AppServicePlanId = appServicePlan.Id,
                AppSettings =
                {
                    { "ApplicationInsights:InstrumentationKey", insightsForSearchUi.InstrumentationKey }
                },
                HttpsOnly = true
            });
        }

        private void RawPriceImportStorage()
        {
            var storageAccount = new Pulumi.Azure.Storage.Account("proxxxrawprice", new Pulumi.Azure.Storage.AccountArgs
            {
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                AccountReplicationType = "LRS",
                AccountTier = "Standard"
            });
        }

        private void AzureSearch()
        {
            var azureSearchService = new Service("proxxx-search", new ServiceArgs
            {
                // Name = "proxxx-search",
                Tags = GetTags(),
                ResourceGroupName = "myresgroup",
                Sku = "basic"
            });
        }

        private void AddSqlServer()
        {
            var sqlServer = new SqlServer("proxxx-dbserver", new SqlServerArgs
            {
                // Name = "proxxx-dbserver",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                AdministratorLogin = "minions",
                AdministratorLoginPassword = "password123!",
                Version = "12.0"
            });

            new Database("proxxx", new DatabaseArgs
            {
                // Name = "proxxx",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                ServerName = sqlServer.Name,
                RequestedServiceObjectiveName = "S0", //S6
                Edition = "Standard",
                MaxSizeGb = "30",
            });

            new Database("PriceImport", new DatabaseArgs
            {
                // Name = "PriceImport",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                ServerName = sqlServer.Name,
                RequestedServiceObjectiveName = "S0",
                Edition = "Standard",
                MaxSizeGb = "10",
            });
        }

        private static InputMap<string> GetTags()
        {
            var inputMap = new InputMap<string>
            {
                {"company", "comp"},
                {"department", "dev"},
            };
            return inputMap;
        }
    }
}

```
Run it then comment in the names and run it again. 

When I logged in into app.pulumi.com I noticed that I could see the whole output. As far as I remember the issue was about an service plan that had a reference to an app service and can't be deleted. Another one was that a resource name already exists.
I think there were other problems too, but I switched over to Azure Storage Backend.

**Comment by Rookian** [comment]
```csharp
using Pulumi;
using Pulumi.Azure.AppInsights;
using Pulumi.Azure.AppService;
using Pulumi.Azure.AppService.Inputs;
using Pulumi.Azure.Search;
using Pulumi.Azure.Sql;

namespace Repro
{
    public class MyStack : Stack
    {
        public MyStack()
        {
            AzureSearch();
            AddSqlServer();
            RawPriceImportStorage();
            AzureSearchApps();
        }

        private void AzureSearchApps()
        {
            var appServicePlan = new Plan("proxxx-pocwebs-win", new PlanArgs
            {
                // Name = "proxxx-pocwebs-win",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                Kind = "App",
                Sku = new PlanSkuArgs
                {
                    Tier = "Basic",
                    Size = "B1",
                }
            });

            
            var insightsForSearchApi = new Insights("proxxxsearchapi", new InsightsArgs
            {
                // Name = "proxxxsearchapi",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                ApplicationType = "web"
            });

            var insightsForSearchUi = new Insights("proxxxprodsearch", new InsightsArgs
            {
                // Name = "proxxxprodsearch",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                ApplicationType = "web"
            });

            new AppService("proxxxsearchapi-poc", new AppServiceArgs
            {
                // Name = "proxxxsearchapi-poc",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                AppServicePlanId = appServicePlan.Id,
                AppSettings =
                {
                    { "ApplicationInsights:InstrumentationKey", insightsForSearchApi.InstrumentationKey }
                },
                HttpsOnly = true
            });

            new AppService("proxxxproductsearch", new AppServiceArgs
            {
                // Name = "proxxxproductsearch",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                AppServicePlanId = appServicePlan.Id,
                AppSettings =
                {
                    { "ApplicationInsights:InstrumentationKey", insightsForSearchUi.InstrumentationKey }
                },
                HttpsOnly = true
            });
        }

        private void RawPriceImportStorage()
        {
            var storageAccount = new Pulumi.Azure.Storage.Account("proxxxrawprice", new Pulumi.Azure.Storage.AccountArgs
            {
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                AccountReplicationType = "LRS",
                AccountTier = "Standard"
            });
        }

        private void AzureSearch()
        {
            var azureSearchService = new Service("proxxx-search", new ServiceArgs
            {
                // Name = "proxxx-search",
                Tags = GetTags(),
                ResourceGroupName = "myresgroup",
                Sku = "basic"
            });
        }

        private void AddSqlServer()
        {
            var sqlServer = new SqlServer("proxxx-dbserver", new SqlServerArgs
            {
                // Name = "proxxx-dbserver",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                AdministratorLogin = "xxx",
                AdministratorLoginPassword = "xxx",
                Version = "12.0"
            });

            new Database("proxxx", new DatabaseArgs
            {
                // Name = "proxxx",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                ServerName = sqlServer.Name,
                RequestedServiceObjectiveName = "S0", //S6
                Edition = "Standard",
                MaxSizeGb = "30",
            });

            new Database("PriceImport", new DatabaseArgs
            {
                // Name = "PriceImport",
                ResourceGroupName = "myresgroup",
                Tags = GetTags(),
                ServerName = sqlServer.Name,
                RequestedServiceObjectiveName = "S0",
                Edition = "Standard",
                MaxSizeGb = "10",
            });
        }

        private static InputMap<string> GetTags()
        {
            var inputMap = new InputMap<string>
            {
                {"company", "comp"},
                {"department", "dev"},
            };
            return inputMap;
        }
    }
}

```
Run it then comment in the names and run it again. 

When I logged in into app.pulumi.com I noticed that I could see the whole output. As far as I remember the issue was about an service plan that had a reference to an app service and can't be deleted. Another one was that a resource name already exists.
I think there were other problems too, but I switched over to Azure Storage Backend.

**Comment by Rookian** [comment]
You can also use a resource group that does not exist.

**Comment by moerketh** [comment]
Did you install Pulumi with Chocolatey? Do you see the same behavior if you install Pulumi manually?

**Comment by Rookian** [comment]
Yes, I did install pulumi with chocolately. I did not test it.

**Comment by leezen** [comment]
@moerketh It does sound a lot like https://github.com/pulumi/pulumi/issues/3275 in that `[409] Conflict: Another update is currently in progress.` suggests a similar problem mentioned in #3275 where there's a hanging process.

**Comment by stack72** [comment]
@Rookian I believe this is fixed via v1.14.0 as I made changes to pulumi-chocolately to not use the built in chocolatey shims

Can you let me know?

**Comment by Rookian** [comment]
> @Rookian I believe this is fixed via v1.14.0 as I made changes to pulumi-chocolately to not use the built in chocolatey shims
> 
> Can you let me know?

Works. Tested with a resource group that does not exist. Now I got the diagnostics view and the process did not hang anymore.
